pyccl/tests/test_tkkssc.py

```diff
@@ -12,7 +13,10 @@
 HBF = ccl.halos.HaloBiasTinker10(COSMO, mass_def=M200)
 P1 = ccl.halos.HaloProfileNFW(ccl.halos.ConcentrationDuffy08(M200),
                               fourier_analytic=True)
+# P2 will have is_number_counts = True
 P2 = ccl.halos.HaloProfileHOD(ccl.halos.ConcentrationDuffy08(M200))
+P2_nogc = ccl.halos.HaloProfileHOD(ccl.halos.ConcentrationDuffy08(M200))
+P2_nogc.is_number_counts = False
```

Comment:
are the integrals the same for NFW than for HOD? If not, I need to use the HOD with `is_number_counts == False` to cheat the code and only compute the first part of the response in a way that it's exactly the same as it will be when I use the HOD (with `is_number_counts == True`